admin/desk/utils.py

```diff
@@ -0,0 +1,194 @@
+"""
+Modified from https://gist.github.com/pamelafox/5855372
+"""
+
+import requests
+import json
+from requests_oauthlib import OAuth1Session
+
+from django.conf import settings
+
+
+class DeskError(Exception):
```

Comment:
It allows us to have a differentiated exception raised rather than raising a generic exception.

Sticking this in `exceptions.py` is not a standard practice. It would be similar to splitting up models and managers into `models.py` and `managers.py`